docs/painless/painless-execute-script.asciidoc

```diff
@@ -0,0 +1,53 @@
+[[painless-execute-api]]
+=== Painless execute API
+
+The Painless execute API allows to execute an arbitrary script and introspect the result.
+
+[[painless-execute-api-parameters]]
+.Parameters
+[options="header"]
+|======
+| Name             | Required  | Default                | Description
+| `script`         | yes       | -                      | The script to execute
```

Comment:
Should this follow something more closely to the stored script format?
```
"script": {
    "code":  "<code>",
    "context": "<context>"
}
```

Reply:
After looking at this again. Changing to format to what you suggest does make the request a bit more verbose. Example: 

```
{
  "script": {
    "code": {
      "source": "params.count / params.total",
      "params": {
        "count": 100.0,
        "total": 1000.0
      }
    },
    "context": ...
  }
}
```

Compared to:


```
POST /_scripts/painless/_execute
{
  "script": {
    "source": "params.count / params.total",
    "params": {
      "count": 100.0,
      "total": 1000.0
    }
  },
  "context": ...
}
```

I don't think this adds much? And `context` is something specific to this api.

Reply:
My apologies, I don't think I made my request clear.  I thought the format was different, what I meant was change "source" to be "code" like we've done for stored scripts, but it looks like we never did that for Script, so it's confusing.  Let's just roll with what you had.